### PR TITLE
Update DM TOC and remove redundant files

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update DM TOC and remove redundant files

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/feature-expression-filter.md [hunk 1]
```diff
@@ -1,91 +1,12 @@
...
-SQL 表达式可以涉及一列或多列，也可使用 TiDB 支持的 SQL 函数，例如 `c % 2 = 0`、`a*a + b*b = c*c`、`ts > NOW()`。
-
-TIMESTAMP 类型的默认时区是 UTC。可以使用 `c_timestamp = '2021-01-01 12:34:56.5678+08:00'` 的方式显式指定时区。
-
-配置项 `expression-filter` 下可以定义多条过滤规则，上游数据源在其 `expression-filters` 配置项中引用需要的规则使其生效。当有多条规则生效时，匹配到**任意一条**规则即会导致某个行变更被过滤。
-
-> **注意：**
->
-> 为某张表设置过多的表达式过滤会增加 DM 的计算开销，可能导致数据迁移速度变慢。
+具体实现细节，请参考 [如何通过 SQL 表达式过滤 DML](/filter-dml-event.md)。
```
Comment (qiancai) [inline dm/feature-expression-filter.md:12]:
```suggestion
具体操作步骤和实现细节，请参考 [如何通过 SQL 表达式过滤 DML](/filter-dml-event.md)。
```

File: dm/quick-create-migration-task.md [hunk 2]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
...
+- [从 Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)
 
-如果你需要将使用分表方案的业务合并迁移到 TiDB，可以参考[分表合并迁移到 TiDB](/dm/usage-scenario-shard-merge.md)。
+## 迁移 MySQL 到 TiDB
 
-## 只迁移数据源增量数据到 TiDB
+如果你没有使用 Cloud storage (S3) 服务，而且网络联通和延迟情况良好，那么从 MySQL 迁移数据到 TiDB 时可以参照下面的方案。
 
-如果你使用其他工具进行了全量数据迁移，例如使用 TiDB Lightning 迁移了全量数据，然后只使用 DM 进行增量数据迁移，那么该场景的数据迁移任务配置可以参考[只迁移数据源增量数据到 TiDB](/dm/usage-scenario-incremental-migration.md)。
+- [从小数据量 MySQL 迁移数据到 TiDB](/migrate-small-mysql-to-tidb.md)
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:32]: itemlist 的链接，建议可以放到正文里。现在这种展示方式，容易把上下的正文和 itemlist 搞混。而且孤立的 itemlist 一般不建议用。
其他几个也都可以考虑修改下。

File: dm/quick-create-migration-task.md [hunk 3]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
...
+如果分表数据总规模特别大（例如大于 1 TiB），并且禁止 TiDB 集群在迁移期间有其他业务写入，那么你可以使用 Lightning 对分表数据进行快速合并导入，然后根据业务需要选择是否使用 DM 进行增量数据 (Binlog) 的分表同步。
+
+- [从大数据量分库分表 MySQL 合并迁移数据到 TiDB](/migrate-large-mysql-shards-to-tidb.md)
+
+## 从文件迁移数据到 TiDB
+
+- [从 CSV 文件迁移数据到 TiDB](/migrate-from-csv-files-to-tidb.md)
+- [从 SQL 文件迁移数据到 TiDB](/migrate-from-sql-files-to-tidb.md)
+
+## 更加复杂的迁移方式
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:53]:
```suggestion
## 复杂的迁移方式
```

File: dm/quick-create-migration-task.md [hunk 4]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
...
+从 Aurora 迁移数据到部署在 AWS 的 TiDB 集群，数据迁移可以分为全量迁移和增量迁移两个步骤进行。请根据你的业务需求选择相应的步骤。
 
-## 分库分表合并迁移到 TiDB
+- [从 Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)
 
-如果你需要将使用分表方案的业务合并迁移到 TiDB，可以参考[分表合并迁移到 TiDB](/dm/usage-scenario-shard-merge.md)。
+## 迁移 MySQL 到 TiDB
 
-## 只迁移数据源增量数据到 TiDB
+如果你没有使用 Cloud storage (S3) 服务，而且网络联通和延迟情况良好，那么从 MySQL 迁移数据到 TiDB 时可以参照下面的方案。
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:30]:
```suggestion
如果你没有使用 Cloud storage (S3) 服务，而且网络联通和延迟情况良好，那么从 MySQL 迁移数据到 TiDB 时，请参考[从小数据量 MySQL 迁移数据到 TiDB](/migrate-small-mysql-to-tidb.md)。
```

File: dm/quick-create-migration-task.md (hunk 2 above)
Comment (hfxsd) [inline dm/quick-create-migration-task.md:32]:
```suggestion
```

File: dm/quick-create-migration-task.md [hunk 5]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
 - 完整的数据迁移任务配置示例，请参考 [DM 任务完整配置文件介绍](/dm/task-configuration-file-full.md)
 - 数据迁移任务的配置向导，请参考 [数据迁移任务配置向导](/dm/dm-task-configuration-guide.md)
 
-## 多数据源汇总迁移到 TiDB
+## 从 Amazon Aurora 迁移数据到 TiDB
 
-如果你需要将多个数据源的数据汇总迁移到 TiDB，此外还需要进行表重命名以防止多个数据源中相同表名在迁移过程中出现冲突，或者需要屏蔽掉某些表的某些 DDL/DML 操作，那么你可以参考[多数据源汇总迁移到 TiDB](/dm/usage-scenario-simple-migration.md)。
+从 Aurora 迁移数据到部署在 AWS 的 TiDB 集群，数据迁移可以分为全量迁移和增量迁移两个步骤进行。请根据你的业务需求选择相应的步骤。
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:24]:
```suggestion
从 Aurora 迁移数据到部署在 AWS 的 TiDB 集群，数据迁移可以分为全量迁移和增量迁移两个步骤进行。具体操作步骤请参考[从 Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)。
```

File: dm/quick-create-migration-task.md [hunk 6]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
...
 
-## 分库分表合并迁移到 TiDB
+- [从 Aurora 迁移数据到 TiDB](/migrate-aurora-to-tidb.md)
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:26]:
```suggestion
```

File: dm/quick-create-migration-task.md [hunk 7]
```diff
@@ -19,18 +19,42 @@ summary: 了解在不同业务需求场景下如何配置数据迁移任务。
...
+- [从 SQL 文件迁移数据到 TiDB](/migrate-from-sql-files-to-tidb.md)
+
+## 更加复杂的迁移方式
+
+DM 在实时同步过程中，多个已有特性可以使得同步过程更加灵活，适应各类业务需求：
+
+- [上游使用 pt/gh-ost 工具的持续同步场景](/migrate-with-pt-ghost.md)
+- [如何过滤 binlog 事件](/filter-binlog-event.md)
+- [如何通过 SQL 表达式过滤 binlog](/filter-dml-event.md)
+- [下游存在更多列的迁移场景](/migrate-with-more-columns-downstream.md)
```
Comment (hfxsd) [inline dm/quick-create-migration-task.md:60]:
```suggestion
- [上游使用 pt/gh-ost 工具的持续同步场景](/migrate-with-pt-ghost.md)
- [下游存在更多列的迁移场景](/migrate-with-more-columns-downstream.md)
- [如何过滤 binlog 事件](/filter-binlog-event.md)
- [如何通过 SQL 表达式过滤 binlog](/filter-dml-event.md)
```

File: TOC.md [hunk 8]
```diff
@@ -256,13 +250,11 @@
...
       - [手动处理 Sharding DDL Lock](/dm/manually-handling-sharding-ddl-locks.md)
+      - [变更同步的 MySQL 实例](/dm/usage-scenario-master-slave-switch.md)
       - [管理迁移表的表结构](/dm/dm-manage-schema.md)
       - [处理告警](/dm/dm-handle-alerts.md)
       - [日常巡检](/dm/dm-daily-check.md)
-    - 使用场景
-      - [从 Aurora 迁移数据到 TiDB](/dm/migrate-from-mysql-aurora.md)
-      - [TiDB 表结构存在更多列的迁移场景](/dm/usage-scenario-downstream-more-columns.md)
-      - [变更同步的 MySQL 实例](/dm/usage-scenario-master-slave-switch.md)
+    - [使用场景](/dm/quick-create-migration-task.md)
```
Comment (hfxsd) [inline TOC.md:257]: 可以考虑挪到 “快速上手” 和 “部署使用” 之间。使用场景是用户比较关心的内容，放到前面会更合理些。

Comment (shichun-0415) [reply TOC.md:257]: @qiancai @sunzhaoyang PTAL. Thanks.

**Review by hfxsd — APPROVED** [review]
LGTM

File: TOC.md (hunk 8 above)
Comment (sunzhaoyang) [reply TOC.md:257]: 有道理，使用场景的子文档都已经被删掉，只留这个文档在顶层目录有点不合适。可以放到“快速上手”下面做个二级目录

File: dm/quick-create-migration-task.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 数据迁移场景概述
+title: 使用场景
 summary: 了解在不同业务需求场景下如何配置数据迁移任务。
 ---
 
-# 数据迁移场景概述
+# 使用场景概述
```
Comment (qiancai) [inline dm/quick-create-migration-task.md:6]:
```suggestion
# DM 使用场景概述
```

**Review by qiancai — APPROVED** [review]

**Review by sunzhaoyang — APPROVED** [review]

**Pull request merged**